src/zope/securitypolicy/metaconfigure.py

```diff
@@ -26,37 +26,43 @@
     principalRoleManager as principal_role_mgr
 
 
-def grant(_context, principal=None, role=None, permission=None):
+def grant(_context, principal=None, role=None, permission=None,
+          permissions=None):
     nspecified = ((principal is not None) +
                   (role is not None) +
-                  (permission is not None))
+                  (permission is not None) +
+                  (permissions is not None))
 
     if nspecified != 2:
         raise ConfigurationError(
-            "Exactly two of the principal, role, and permission attributes "
-            "must be specified")
+            "Exactly two of the principal, role, and permission resp. "
+            "permissions attributes must be specified")
```

Comment:
Does not sound clear enough to me. I'm not a native english speaker, but what about this: "Exactlty two out of these arguments must be specified: principal, role and permission or permissions."?